### ISSUE TITLE
Thumb carpometacarpal (CMC) subluxation or dislocation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11035]
**Preferred term label:**
Thumb carpometacarpal (CMC) subluxation or dislocation

**Synonyms**

**Definition (free text, please give PubMed ID)**
Thumb radiographs showing CMC joint space widening and/or incongruity of the joint space between thumb metacarpal bone and the trapezium bone

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
- Thumb carpometacarpal (CMC) dislocation
- Basilar thumb arthritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001264
- label: Thumb carpometacarpal subluxation
- parents: [Abnormal thumb morphology (HP:0001172)](https://hpo.jax.org/app/browse/term/HP:0001172)
- pmid: PMID:33927510
- def: A partial (subluxation) loss of alignment at the base of the thumb, where the first metacarpal bone meets the trapezium bone of the wrist. Thumb radiographs showing carpometacarpal (CMC) joint space widening and/or incongruity of the joint space between thumb metacarpal bone and the trapezium bone.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11035
- synonyms: Thumb carpometacarpal dislocation(NONE)|Thumb CMC subluxation(NONE)
